dm/dm-compatibility-catalog.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Compatibility Catalog of TiDB Data Migration
+summary: This document describes the compatibility between DM of different versions and upstream/downstream databases.
+---
+
+# Compatibility Catalog of TiDB Data Migration
+
+DM supports migrating data from different sources to TiDB clusters. Based on the data source type, DM has four compatibility levels:
+
+- **Generally available (GA)**: The application scenario has been verified and passed the GA test.
+- **Experimental**: Although the application scenario has been verified, the test does not cover all scenarios or involves only a limited number of users. The application scenario might encounter problems occasionally.
+- **Not tested**: DM is expected to be always compatible with MySQL during iteration. However, due to resource constraints, we cannot test all MySQL-derived versions. Therefore, the application is technically compatible with MySQL, but is not fully tested, which means you need to perform a test to ensure compatibility.
+- **Incompatible**: DM is proved to be incompatible with the data source and the application is not recommended for use in production environments.
+
+## Data sources
+
+|Data source|Compatibility level|Remarks|
+|-|-|-|
+|MySQL ≤ 5.5|Not tested||
+|MySQL 5.6|GA||
+|MySQL 5.7|GA||
+|MySQL 8.0|Experimental||
+|MariaDB < 10.1.2|Incompatible|Incompatible with binlog of the time type|
+|MariaDB 10.1.2 ~ 10.5.10|Experimental||
+|MariaDB > 10.5.10|Incompatible|Permission errors reported in the check procedure|
+
+## Target databases
+
+> **Warning:**
+>
+> DM 5.3.0 is not recommended because data replication fails with low probability if you have enabled GTID replication but do not enable relay log.
```

Comment:
It's recommended to put if clauses upfront. [ref](https://developers.google.com/style/clause-order)
```suggestion
> DM v5.3.0 is not recommended. If you have enabled GTID replication but do not enable relay log in DM v5.3.0, data replication fails with low probability.
```